pyinstaller.spec

```diff
@@ -15,17 +15,6 @@ if not hasattr(sys, 'real_prefix'):
     sys.exit("Please run inside a virtualenv with Tahoe-LAFS installed.")
 
 
-# Ugly hack to disable the setuptools requirement asserted in '_auto_deps.py'.
```

Comment:
I'm glad to see this go :). Could you also take a look in tox.ini, the "pyinstaller" environment has a comment about handling this source-tree patching. Maybe see if you can remove that comment (and whatever weirdness it describes, maybe the `usedevelop` and `deps = .` lines) along with this PR? No big deal if not, if `tox -e pyinstaller` works, I'm happy to leave it alone.

Reply:
I cleaned up the tox testenv accordingly (and added the `packaging` dep -- which now appears to be required after the "setuptools>=28" changes..).

Reply:
Cool, thanks. Can we also remove the `deps = .` line? I'm thinking that it was an artifact of using `skipinstall`, and is no longer necessary.